app/frontend/src/components/base/BaseActionCard.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <router-link class="action-link" :to="{ name: linkName, params: linkParams }">
+    <v-card class="action-card" max-width="344">
```

Comment:
Is there any way to make the max-width relative instead? It may be better to have the card itself not define how large itself is, and let the calling component decide instead based on how big the parent div is

Reply:
Yeah that was hanging around, not needed (will indeed use parent size to determine).
Removed